Tests/Engine/CustomizedRule.tests.ps1

```diff
@@ -113,10 +113,10 @@ Describe "Test importing correct customized rules" {
 		It "will show the custom rules when given glob with recurse switch" {
 			# needs fixing for Linux
 			$expectedNumRules = 5
-			if ($IsLinux)
-			{
-				$expectedNumRules = 4
-			}
+			#if ($IsLinux)
+			#{
+			#	$expectedNumRules = 4
+			#}
```

Comment:
We should just delete those commented out code lines now that it works without a special case (not sure why, I guess some upstream dependency received a fix)